lxd/storage/block/utils.go

```diff
@@ -0,0 +1,39 @@
+package block
+
+import (
+	"os"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/canonical/lxd/shared"
+)
+
+// BlockDiskSizeBytes returns the size of a block disk (path can be either block device or raw file).
+func BlockDiskSizeBytes(blockDiskPath string) (int64, error) {
```

Comment:
Should be called DiskSizeBytes to avoid stammering when doing `block.BlockDiskSizeBytes`.

Reply:
Done, thanks.